terminal.py

```diff
@@ -5,21 +5,10 @@
 import argparse
 import os
 import sys
-
-from prompt_toolkit.completion import NestedCompleter
-
-from gamestonk_terminal import config_terminal
-from gamestonk_terminal import feature_flags as gtff
-from gamestonk_terminal.helper_funcs import get_flair
-from gamestonk_terminal.menu import session
-from gamestonk_terminal.terminal_helper import (
-    about_us,
-    bootup,
-    check_api_keys,
-    print_goodbye,
-    reset,
-    update_terminal,
-)
```

Comment:
Moving imports from the top of the file to code internals decreases readability of the code.